server/service/devices.go

```diff
@@ -335,3 +337,79 @@ func transparencyURL(ctx context.Context, request interface{}, svc fleet.Service
 
 	return transparencyURLResponse{RedirectURL: transparencyURL}, nil
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// Get Current Device's MDM Apple Enrollment Profile
+////////////////////////////////////////////////////////////////////////////////
+
+type getDeviceMDMManualEnrollProfileRequest struct {
+	Token string `url:"token"`
+}
+
+func (r *getDeviceMDMManualEnrollProfileRequest) deviceAuthToken() string {
+	return r.Token
+}
+
+type getDeviceMDMManualEnrollProfileResponse struct {
+	// Profile field is used in hijackRender for the response.
+	Profile []byte
```

Comment:
should this be unexported?

Reply:
The type is unexported, so that's fine regarding use outside the package. The field itself would not need to be, but for consistency with other response structs I exported it (this is required for JSON-marshaled responses).